docs/source/install.rst

```diff
@@ -154,7 +154,7 @@ These optional dependencies and their minimum supported versions are listed belo
 +------------------+-----------------+---------------------------------------------------------------------------------------------------------+
 | `scipy`_         | ``>=1.7.2``     | Required for ``dask.array.stats``, ``dask.array.fft``, and :func:`dask.array.linalg.lu`                 |
 +------------------+-----------------+---------------------------------------------------------------------------------------------------------+
-| `sparse`_        | ``>=0.12.0``    | Use sparse arrays as backend for dask arrays                                                            |
+| `sparse`_        | ``>=0.13.0``    | Use sparse arrays as backend for dask arrays                                                            |
```

Comment:
This page also needs to update numpy and pyarrow versions, but fine as a follow up